nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/FeaturesRunner.java

```diff
@@ -132,10 +132,13 @@ public URL getTargetTestResource(String name) throws IOException {
      * @since 5.7
      */
     public <T extends Annotation> T getConfig(FrameworkMethod method, Class<T> type) {
-        T config = method.getAnnotation(type);
-        if (config != null) {
-            return config;
+        if (method != null) {
```

Comment:
yep, this happened see [here](https://github.com/nuxeo/nuxeo/blob/8edcdc7b1d597136ccff5494f902253a51acc6ef/nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/LogFeature.java#L222) most of the time i made one method which will be called in the both cases (Class Or Method). To avoid controlling on method param in the logFeature i took the choice to put the control in base class.